README.md

```diff
@@ -10,51 +10,36 @@ We recommend including the .d.ts file by downloading this package through [npm](
 ```bash
 
 # Install types and save in package.json as a development dependency
-npm install @microsoft/microsoft-graph-types --save-dev
+npm install @microsoft/microsoft-graph-types-beta --save-dev
 
 ```
 
-
-![GIF showing intellisense and autocompletion for Microsoft Graph entities in Visual Studio Code ](https://github.com/microsoftgraph/msgraph-typescript-typings/raw/master/typings-demo.gif)
+![GIF showing intellisense and autocompletion for Microsoft Graph entities in Visual Studio Code ](https://github.com/microsoftgraph/msgraph-beta-typescript-typings/blob/main/typings-demo.gif)
 ## Examples
-The following examples assume that you have a valid access token. We used [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch) to perform requests, but you can use [our JavaScript client library](https://github.com/microsoftgraph/msgraph-sdk-javascript) or other libraries as well.
-```typescript
-import * as MicrosoftGraph from "@microsoft/microsoft-graph-types"
+The following examples assume that you have a valid access token. The following example uses [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch) and  [Microsoft Graph JavaScript client library](https://github.com/microsoftgraph/msgraph-sdk-javascript) -
 
-import * from 'isomorphic-fetch';
-const accessToken:string = "";
-```
-### List my recent messages
 ```typescript
-let url = "https://graph.microsoft.com/v1.0/me/messages";
-let request = new Request(url, {
-    method: "GET",
-    headers: new Headers({
-        "Authorization": "Bearer " + accessToken
-    })
-});
-
-fetch(request)
-.then((response) => {
-    response.json().then((res) => {
-        let messages:[MicrosoftGraph.Message] = res.value;
-        for (let msg of messages) { //iterate through the recent messages
-            console.log(msg.subject);
-            console.log(msg.toRecipients[0].emailAddress.address);
-        }
-    });
+import { User } from "@microsoft/microsoft-graph-types-beta";
+
+import { Client } from "@microsoft/microsoft-graph-client";
 
-})
-.catch((error) => {
-    console.error(error);
-});
+import 'isomorphic-fetch';
+
+const client = Client.initWithMiddleware({
+		debugLogging: true,
+		middleware,
+	});
+
+const response = await client.api("/me").get();
+const user = response as User;
 ```
-### Send an email as the logged in user
+
+### Example of creating a object
 ```typescript
 // Create the message object
 
 // Note that all the properties must follow the interface definitions.
-// For example, this will not compile if you try to type "xml" instead of "html" for contentType. 
+// For example, this will not compile if you try to type "xml" instead of "html" for contentType.
 
 let mail:MicrosoftGraph.Message = {
```

Comment:
Do we expect the namespace to be `MicrosoftGraph` or `MicrosoftGraphBeta`?

Reply:
It would be very helpful to have a beta namespace.

Reply:
```
import * as MicrosoftGraph from "@microsoft/microsoft-graph-types-beta"
const user: MicrosoftGraph.User = null; 
```
Something like this works too. 
I have updated this to `let mail:MicrosoftGraphBeta.Message` for clarity.